Sources/ArgumentParser/Parsable Types/CommandConfiguration.swift

```diff
@@ -45,6 +45,9 @@ public struct CommandConfiguration {
   /// A Boolean value indicating whether this command should be shown in
   /// the extended help display.
   public var shouldDisplay: Bool
+
+  /// A Boolean value indicating whether to prompt for missing required values.
+  public var isInteractable: Bool
```

Comment:
I don't think we're quite there yet with `isInteractable` — what other names have you considered? This is a fairly small API, especially if changing the default will hopefully be rare, so maybe something longer like `shouldPromptForMissing`?